forms-flow-web/src/components/Form/List.js

```diff
@@ -151,7 +151,13 @@ const List = React.memo((props) => {
     searchText,
     formType
   ]);
-
+  useEffect(() => {
+    setIsSearchValid(searchValidator(searchTextInput) === false
```

Comment:
can you come up with a better implementation? better `searchValidator` return a boolean and passes all responsibility to the same function so you can avoid following lines of code.